tracer/test/snapshots/AspNetCoreMvc31Tests.NoFF.__path=_handled-exception_statusCode=500.verified.txt

```diff
@@ -2,20 +2,52 @@
   {
     TraceId: Id_1,
     SpanId: Id_2,
-    Name: aspnet_core.request,
-    Resource: GET handled-exception,
+    Name: custom_post_routing,
+    Resource: custom_post_routing,
```

Comment:
I don't _quite_ understand why the exception is caught by the `custom_post_routing` middleware... May need to look into that one unless someone can see why?